### PR TITLE
Create mood submit toast popup message

### PR DESCRIPTION
**Pull request by yasir-satti**
This is the code and test for the toast popup messgae

### CODE REVIEW

File: KotlinMMP/iosApp/.gitignore [hunk 1]
```diff
@@ -12,6 +12,8 @@ build/
 dist/
 node_modules/
 aws-exports.js
+awsconfiguration.json
+amplifyconfiguration.json
```
Comment (cesimcivisilli) [inline KotlinMMP/iosApp/.gitignore:16]: we don't need these to be in .gitignore, can you remove?

File: KotlinMMP/iosApp/iosApp/ContentView.swift [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import SwiftUI
 import shared
 import Amplify
+import SSToastMessage
 
 struct ContentView: View {
     @State private var selectedTab = MoodService().getMood(id: 0).moodValue
+    @State var showToast = false
+    
+//    let moodSubmitToastColor: UIColor = UIColor(named: "MoodSubmitToastColor")!
```
Comment (cesimcivisilli) [inline KotlinMMP/iosApp/iosApp/ContentView.swift:10]: remove comment if not needed?

File: KotlinMMP/iosApp/iosApp/iOSApp.swift [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import SwiftUI
-import AppCenter
-import AppCenterAnalytics
-import AppCenterCrashes
+//import AppCenter
+//import AppCenterAnalytics
+//import AppCenterCrashes
```
Comment (cesimcivisilli) [inline KotlinMMP/iosApp/iosApp/iOSApp.swift:4]: why were these commented out?

File: KotlinMMP/iosApp/iosApp/iOSApp.swift [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 
 @main
 struct iOSApp: App {
     init() {
-        AppCenter.start(withAppSecret: "388e8513-eaaf-4c7d-8bca-dfcddcc20128", services: [Analytics.self, Crashes.self])
+//        AppCenter.start(withAppSecret: "388e8513-eaaf-4c7d-8bca-dfcddcc20128", services: [Analytics.self, Crashes.self])
```
Comment (cesimcivisilli) [inline KotlinMMP/iosApp/iosApp/iOSApp.swift:9]: same as above, why was this commented out?

**Review by MariliaE — COMMENTED** [review]
There are a lot of Amplify changes that shouldn't be pushed. The amplify folder was deleted from the root of the project but it looks like it's being added again through this PR probably from local changes, can you please check that the files changed for this PR are only related to the snackbar work? If you need any help with what files should be removed happy to jump on a call

**Review by MariliaE — APPROVED** [review]

**Pull request merged**